### PR TITLE
added d64-66

### LINKED ISSUE #30: 90 ngày devops day 65 translation

### LINKED ISSUE #32: 90 ngày DevOps day 66 translation

### PR DESCRIPTION
**Pull request by qmauvnt**
resolve #30 
resolve #32

### CODE REVIEW

**Comment by qmauvnt** [comment]
@hungran @haicasgox 2 sếp review giúp em nhé!

File: 2022/vi/Days/day64.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+tags: 'devops, 90daysofdevops, learning'
+cover_image: null
+canonical_url: null
+id: 1048765
+---
+
+
+## Ansible: Bắt đầu
+
+Chúng ta đã đề cập một chút về Ansible là gì trong [bài viết ngày hôm qua](day63.md) Nhưng chúng ta sẽ có một số thông tin bổ sung ở đây. Thứ nhất, Ansible là sản phẩm của RedHat. Thứ hai, agentless , kết nối thông qua SSH và chạy các câu lệnh. Thứ ba, nó đa nền tảng (Linux & macOS, WSL2) và là mã nguồn mở (cũng có tùy chọn trả phí cho doanh nghiệp).
```
Comment (hungran) [inline 2022/vi/Days/day64.md:14]:
```suggestion
Chúng ta đã đề cập một chút về Ansible là gì trong [bài viết ngày hôm qua](day63.md) Nhưng chúng ta sẽ có một số thông tin bổ sung ở đây. Thứ nhất, Ansible là sản phẩm của RedHat. Thứ hai, agentless, kết nối thông qua SSH và chạy các câu lệnh. Thứ ba, nó đa nền tảng (Linux & macOS, WSL2) và là mã nguồn mở (cũng có tùy chọn trả phí cho doanh nghiệp).
```

File: 2022/vi/Days/day64.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+Hoặc cách sử dụng thực tế cho một module có thể là `ansible webservers -m service -a "name=httpd state=started"` điều này sẽ cho chúng tôi biết liệu tất cả máy chủ web của chúng tôi có dịch vụ httpd đang chạy hay không. Thuật ngữ máy chủ web được sử dụng trong lệnh đó.
+
+### hosts
+
+Cách tôi sử dụng máy chủ cục bộ ở trên để chạy module ping trên hệ thống, tôi không thể chỉ định một máy khác trên mạng của mình, ví dụ: trong môi trường tôi đang sử dụng máy chủ Windows nơi VirtualBox đang chạy có bộ điều hợp mạng với IP 10.0.0.1 nhưng bạn có thể thấy bên dưới rằng tôi có thể ping tới nó nhưng tôi không thể sử dụng ansible để thực hiện tác vụ đó.
+
+![](../../Days/Images/Day64_config3.png)
+
+Để chúng ta chỉ định máy chủ của mình hoặc các node mà chúng ta muốn tự động hóa với các tác vụ này, chúng ttaôi cần xác định chúng. Chúng ta có thể xác định chúng bằng cách điều hướng đến thư mục /etc/ansible trên hệ thống của bạn.
```
Comment (hungran) [inline 2022/vi/Days/day64.md:49]:
```suggestion
Để chúng ta chỉ định máy chủ của mình hoặc các node mà chúng ta muốn tự động hóa với các tác vụ này, chúng ta cần xác định chúng. Chúng ta có thể xác định chúng bằng cách điều hướng đến thư mục /etc/ansible trên hệ thống của bạn.
```

File: 2022/vi/Days/day64.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+![](../../Days/Images/Day64_config8.png)
+
+Sau đó, chúng ta có các yêu cầu về các nodes, đây là những hệ thống đích mà bạn muốn tự động hóa cấu hình. Chúng ta không cài đặt bất kỳ thứ gì cho Ansible trên những hệ thống này (ý tôi là chúng ta có thể đang cài đặt phần mềm nhưng không có ứng dụng khách nào từ Ansible mà chúng ta cần) Ansible sẽ thực hiện kết nối qua SSH và gửi tất cả mọi thứ qua qua SFTP.(Nếu bạn muốn và bạn đã định cấu hình SSH, bạn có thể sử dụng SCP và SFTP.)
+
+### Các câu lệnh Ansible
+
+Bạn đã thấy rằng chúng ta có thể chạy `ansible Linux -m ping` trên máy Linux của chúng ta và nhận được phản hồi, về cơ bản, với Ansible, chúng ta có thể chạy nhiều lệnh đặc biệt. [ad hoc commands](https://docs.ansible.com/ansible/latest/user_guide/intro_adhoc.html)
+
+You saw that we were able to run `ansible Linux -m ping` against our Linux machine and get a response, basically, with Ansible we can run many ad-hoc commands. Nhưng bạn có thể chạy chương trình này với một nhóm hệ thống và lấy lại thông tin đó. [ad hoc commands](https://docs.ansible.com/ansible/latest/user_guide/intro_adhoc.html)
```
Comment (hungran) [inline 2022/vi/Days/day64.md:75]:
```suggestion
```

**Review by hungran — CHANGES_REQUESTED** [review]

**Comment by qmauvnt** [comment]
Thanks for the review @hungran. Updated as suggested. Could you please review it again?

**Review by hungran — APPROVED** [review]
LGTM

**Pull request merged**